prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -27,6 +29,153 @@ def check_for_location_conflicts(substations):
         )
 
 
+def map_lines_to_substations_using_coords(substations, lines, **kwargs):
+    """Map lines to substations using coordinates
+
+    :param pandas.DataFrame substations: data frame of substations.
+    :param pandas.DataFrame lines: data frame of all lines.
+    :param int \\*\\*kwargs: optional arguments.
+    :return: (*pandas.DataFrame*) -- indices: lines id, columns: lines endpoint
+        coordinates (*FROM* and *TO*) and id of substations found at these coordinates (*FROM_SUB* and *TO_SUB*)
```

Comment:
Should we cut this long line off?

Reply:
I can but it still fits within the 89 character length (enforce by `black` for the code but not the docstring)

Reply:
I shortened the docstrings